app/src/main/java/com/github/wnder/User.java

```diff
@@ -1,10 +1,37 @@
 package com.github.wnder;
 
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
 import android.net.Uri;
+import android.os.Looper;
+import android.util.Log;
 
-public interface User {
+import androidx.annotation.NonNull;
+import androidx.core.app.ActivityCompat;
 
-    public String getName();
+import com.google.android.gms.maps.model.LatLng;
```

Comment:
Same as above in MainActivity.
Since we're using MapBox wouldn't it be more logic to use the MapBox import for LatLng?